### PR TITLE
React Hooks Example

### PR DESCRIPTION
**Pull request by captDaylight**
This took me a second to work through how it works with React hooks, might save someone some time in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,27 @@ import * as typeformEmbed from '@typeform/embed'
 const typeformEmbed = require('@typeform/embed')
 ```
 
+#### With React Hooks
+```jsx
+import React, { useRef, useEffect } from 'react';
+
```
Comment (mathio) [inline README.md:53]: Your code says `typeformEmbed is not defined`. You need to import it.
```suggestion
import * as typeformEmbed from "@typeform/embed";

```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,27 @@ import * as typeformEmbed from '@typeform/embed'
...
+const MyTypeform = () => {
+  const typeformRef = useRef(null);
+  
+  useEffect(() => {
+    typeformEmbed.makeWidget(typeFormRef.current, 'https://form.typeform.com/to/MY_TYPEFORM_ID', {
```
Comment (mathio) [inline README.md:58]: `typeFormRef is not defined` - you have a typo in the variable name (uppercase F).

```suggestion
    typeformEmbed.makeWidget(typeformRef.current, 'https://form.typeform.com/to/MY_TYPEFORM_ID', {
```

**Review by mathio — CHANGES_REQUESTED** [review]
Thank you for your PR. Please see my comments to make the code runnable just by copy-pasting.

**Comment by captDaylight** [comment]
Sorry for the late reply, glad to see it make its way in!

**Pull request merged**